src/unstract/sdk/adapters/embedding/no_op_embedding/src/static/json_schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "NoOp Embedding Adapter",
+  "type": "object",
+  "required": [
+    "adapter_name",
+    "wait_time"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "Name",
+      "default": "",
+      "description": "Provide a unique name for this No Op adapter instance. Example: no-op-instance-1"
+    },
+    "wait_time": {
+      "type": "number",
+      "title": "Wait time",
+      "default": "0",
+      "description": "Provide the wait time."
```

Comment:
```suggestion
      "description": "Provide the time to wait before returning the response"
```

Reply:
@harini-venkataraman Please mention the unit as well in the title and description. In this case it is seconds.